routes/posts_routes.js

```diff
@@ -0,0 +1,10 @@
+
+const express = require('express');
+const router = express.Router();
+const Post = require('../controllers/post');
+
```

Comment:
Please fix this typo of renaming posts controller:

```suggestion
const Post = require('../controllers/posts_controller');
```